slumber.yml

```
@@ -37,6 +37,14 @@ chains:
         - baz
         - a really really really really long option
         - "{{chains.username}}"
+  dynamic_options:
+    source: !request
+      recipe: login
+    selector: $.args.fast
```

Including the full `args` will show off how it handles nested arrays
```suggestion
    selector: $.args[*]
```
Also I just noticed this won't work on a fresh DB, because `login`'s recipe is self-referential. With no request history, we can't create new requests. Maybe move this chain to be a query param for `get_user`?